README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+It is the basic structure of a website.
```

Comment:
Semantic code help s the browser understand the code for accessibility readers and SEO as well as allowing others to understand the format of your code.